apps/auction/src/pages/home/Home.tsx

```diff
@@ -0,0 +1,32 @@
+import { Loader } from 'components';
+import { ADDRESS } from 'consts';
+import { useAuction, useNft, useStatus } from 'hooks';
+import { Buy } from './buy';
+import { Sell } from './sell';
+import { Start } from './start';
+
+function Home() {
+  const auction = useAuction();
+  const { nftContractActorId, tokenId } = auction || {};
+
+  const { nft, isNftStateRead } = useNft(nftContractActorId, tokenId);
+  const { media } = nft || {};
+  const src = `${ADDRESS.IPFS_GATEWAY_ADDRESS}/${media}`;
+
+  const { status, resetStatus, setExpiredStatus, setRunningStatus } = useStatus(auction?.status);
+  const purchasePrice = typeof status === 'object' ? status.Purchased.price : undefined;
```

Comment:
status?.Purchased.price

Reply:
if `status` is object, these props gonna be there for sure

Reply:
use es6 syntax